src/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface Todos {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
+
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+export interface TodoWithUser extends Todos {
+  name: string;
+  username: string;
+  email: string;
+}
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  name: string;
+  email: string;
+}
```

Comment:
The `Todo` interface includes `name` and `email` properties, which are not part of a typical todo item. If these are intended to be part of `TodoWithUser`, ensure that the `Todo` interface accurately reflects the structure of a todo item without user details.